gno.land/pkg/gnoclient/client_txs.go

```diff
@@ -81,6 +98,78 @@ func (c *Client) Call(cfg CallCfg) (*ctypes.ResultBroadcastTxCommit, error) {
 	return c.signAndBroadcastTxCommit(tx, accountNumber, sequenceNumber)
 }
 
+// MultiCall executes a contract call on the blockchain.
+func (c *Client) MultiCall(cfg MultiCallCfg) (*ctypes.ResultBroadcastTxCommit, error) {
+	// Validate required client fields.
+	if err := c.validateSigner(); err != nil {
+		return nil, errors.Wrap(err, "validate signer")
+	}
+	if err := c.validateRPCClient(); err != nil {
+		return nil, errors.Wrap(err, "validate RPC client")
+	}
+
+	sequenceNumber := cfg.SequenceNumber
+	accountNumber := cfg.AccountNumber
+
+	msgs := make([]vm.MsgCall, 0, len(cfg.Msgs))
+	for _, msg := range cfg.Msgs {
+		pkgPath := msg.PkgPath
+		funcName := msg.FuncName
+		args := msg.Args
+		send := msg.Send
+
+		// Validate config.
+		if pkgPath == "" {
+			return nil, errors.New("missing PkgPath")
+		}
+		if funcName == "" {
+			return nil, errors.New("missing FuncName")
+		}
```

Comment:
These errors should be extracted out as package-level

Reply:
[9a5fd71](https://github.com/gnolang/gno/pull/1565/commits/9a5fd71c6251e16887da7dbcc7ef4d6c04e98993)